examples/fluids/smartsim_regression_framework.py

```diff
@@ -79,6 +79,9 @@ def test(self, ceed_resource) -> Tuple[bool, Exception, str]:
                 '-degree', '1',
                 '-sgs_train_enable',
                 '-sgs_train_put_tensor_interval', '2',
+                '-bc_symmetry_z', '',
```

Comment:
What is the empty argument for?

Reply:
The `blasius.yaml` file specifies the bc_symmetry faces, but this is to unset them. The empty argument is necessary, but is there to clarify what it's doing (setting those faces to nothing). I can remove it if you want.

Reply:
I suppose that's okay. It's unusual to make an zero-length argument on the command line, which is why it looked odd to my eye.

Reply:
I've just gone ahead and removed the extra argument.